charts/mirrorbits/tests/custom_values_test.yaml

```diff
@@ -12,8 +34,6 @@ templates:
 tests:
   - it: Should set the correct service selector labels when a fullNameOverride is specified
     template: service.files.yaml
-    set:
-      fullNameOverride: "my-fullNameOverride"
```

Comment:
I don't understand why this set has been removed, this is what this test is testing 🤔

Reply:
read the diff. it has been moved on top.

Reply:
Changed as per your review feedbacks